lib/src/date-input/DateInput.stories.tsx

```diff
@@ -108,9 +114,9 @@ export const Chromatic = () => (
 );
 
 const DatePicker = () => (
```

Comment:
I think the example 'Show Date Picker' can be avoided. If we add the regular Date Picker to the story 'Date Picker Button States' and apply the same play function to this first example we would be testing the same.

![image](https://user-images.githubusercontent.com/44321109/214833225-6c91b3f1-2556-4760-8b4d-bd4e7f73f148.png)

To keep testing the distance between the date picker and the text input you can add a click example to the first story 'Chromatic'.

Reply:
These changes were not done as I mentioned so I reopen it.